theano/misc/check_blas.py

```diff
@@ -272,6 +272,7 @@ def test():
         GTX 660           2.32s  2.32s
         GTX 580           2.42s         2.47s
         GTX 480           2.87s         2.88s
+        TX1                      7.6s
```

Comment:
This is in float32, not float16. Theano did not implement computation in
float16.

On Tue, Jan 26, 2016 at 4:41 AM, Titus Nicolae <notifications@github.com>
wrote:

> In theano/misc/check_blas.py
> <https://github.com/Theano/Theano/pull/3918#discussion_r50812419>:
>
> > @@ -272,6 +272,7 @@ def test():
> >          GTX 660           2.32s  2.32s
> >          GTX 580           2.42s         2.47s
> >          GTX 480           2.87s         2.88s
> > +        TX1                      7.6s
>
> is this comparison legitimate since TX1 does FP16 not FP32?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3918/files#r50812419>.
>